map_app/static/js/data_processing.js

```diff
@@ -27,6 +31,10 @@ async function subset() {
 
 
 async function subset_to_file() {
+    if (Object.keys(wb_id_dict).length === 0) {
+        alert('Please select at least one item before subsetting');
```

Comment:
Ditto above: 
Suggest: "Please select at least one basin in the map before subsetting"